spec/migrations/20151021104529_add_persistent_volumes_to_container_volumes_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+require_migration
+
+describe AddPersistentVolumesToContainerVolumes do
+  let(:container_volume_stub)  { migration_stub(:ContainerVolume) }
+
+  migration_context :up do
+    it "up" do
+      container_volume = container_volume_stub.create!
+
+      migrate
+
+      expect(container_volume.reload).to have_attributes(:parent_type => "ContainerGroup")
+    end
+  end
+
+  migration_context :down do
+    it "down" do
+      container_volume = container_volume_stub.create!(:parent_type => "ContainerGroup")
```

Comment:
Please also add a:

    persistent_volume = container_volume_stub.create!(:parent_type => "(...put the relevant class here...)")

and then check that it was deleted.